introduction.md

```diff
@@ -1,2 +1,4 @@
-Hello, I'm Disco.  My favourite Code School path has been Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?